README.md

```diff
@@ -1,20 +1,22 @@
 ## sqlcommenter
 
+**sqlcommenter is donated to [OpenTelemetry](https://opentelemetry.io/). Please refer to <https://github.com/open-telemetry/opentelemetry-sqlcommenter> for futher updates.**
```

Comment:
It seems better to copy most of the text from the end of this comment: [1](https://github.com/open-telemetry/opentelemetry-java-contrib/issues/205#issuecomment-1018136934), 

```
We will continue to maintain this repository until opentelemetry SQLCommenter feature stabilizes. 
Further updates will be added here in due time.
```

The reason i don't want to add link to [opentelemetry-sqlcommenter](https://github.com/open-telemetry/opentelemetry-sqlcommenter) here is because it can confuse users.

Reply:
Thanks for the comment! Do you mean we don't put hyperlink to opentelemetry-sqlcommenter repository regardless of the reference to "opentelemetry SQLCommenter"?

Reply:
Yes, as that(opentelemetry-sqlcommenter) is an interim repository and not meant to go live really, it might confuse users if they go there and see similar code and start to use it.

By `Opentelemetry SQLCommenter feature` i meant it as a feature which will stabilize when we merge with all opentelemetry language and contrib libraries. 

Instead of doing this:
google/sqlcommenter --> opentelemetry-sqlcommenter --> future opentelemetry-contrib
we can do this instead 
google/sqlcommenter ---> (in future we will give the correct links to opentelemetry contrib)

What do you think ?

Reply:
@sjs994 sorry for super delayed comment. I wasn't aware of your feedback until I cleaned up my GitHub inbox today. Now I changed the comment.